config/schedule.rb

```diff
@@ -0,0 +1,3 @@
+every 1.day, at: '00:00' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.